lib/model/maalcie/MaaltijdAbonnementenModel.class.php

```diff
@@ -220,67 +199,32 @@ public static function inschakelenAbonnementVoorNovieten($mrid) {
 	 * @return MaaltijdAbonnement[]|int OR aantal nieuwe abonnementen novieten
 	 * @throws Exception
 	 */
-	private static function newAbonnement($mrid, $uid = null) {
```

Comment:
Jep, ik heb er twee functies van gemaakt, omdat een abo maken gebruikt kan worden in het aanmelden van alle novieten.